lang/en/docs/tutorials/dft/chemical/molecule-surface.md

```diff
@@ -0,0 +1,89 @@
+
+## Combine the Two Materials
+
+The Multi-Materials 3D Editor allows the two materials under investigation, the benzene molecule and the gold surface, to be **combined** together into a new unified materials entity. 
+
+Care should be taken by the user to place the molecule on top of the surface in as "symmetrical" a way as possible, for example by positioning the center of the benzene ring on the central portion of the surface. Relocation of the benzene molecule position can be done by following the instructions contained [in this page](../../../materials-designer/3d-editor/editor-actions/move-rotate-atoms.md), after selecting the benzene atom components under the ["Scene" sidebar list](../../../materials-designer/3d-editor/edit.md#3.-scene) of the 3D Editor interface.
+
+Since in this example the plane of the 2D benzene molecule and the gold (211) surface are already parallel to each other, a simple [translation](../../../materials-designer/3d-editor/editor-actions/move-rotate-atoms.md#translation) of the benzene atoms on top of the surface should suffice.
+
+## Exit Multi-Materials 3D Editor
+
+After the correct desired positioning of the benzene molecule on top of the gold surface, the user should now [exit](../../../materials-designer/3d-editor/edit.md#exit-the-editor) the Multi-Materials 3D Editor, and return to the original Materials Designer interface.
+
+The user will notice that a new material entry, called "New Material" by default, has now been created automatically and is listed within the [left-had items list sidebar](../../../materials-designer/sidebar-items.md) of the Materials Designer interface. It contains the combined benzene-gold surface crystallographic structure, as a new single material entity.
+
+This new entry should first be [renamed](../../../materials-designer/sidebar-items.md#edit-name-of-item) to a more memorable form, and should finally be [saved](../../../materials-designer/header-menu/input-output/save.md) via the ["Input/Output" Menu](../../../materials-designer/header-menu/input-output.md) located at the top-left corner into the account-owned materials [collection](../../../accounts/collections.md), as a new material structure entry which is distinct from both the original isolated benzene molecule and gold structure.
+
+## Run Further Reaction Profile Calculations
+
+The user is now free to run reaction energy profile calculations on the newly generated benzene-gold surface system, in order to calculate the size of the corresponding activation energy barrier for a chemical adsorption process of the molecule into the surface for instance. 
```

Comment:
The user is now free to use the newly generated benzene-gold surface system, in order to perform its further analysis, such as studying the adsorption energy.